bin/travis_setup.sh

```diff
@@ -22,9 +22,11 @@ else
     sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/test
     sudo apt-get -qq update
     sudo apt-get install -y -qq \
+      gcc-5 g++-5 \
```

Comment:
Adding more dependencies to the core is really hard to justify. Every single additional dependency makes it harder to setup environment on older Linux distributions.